pandera/api/pandas/container.py

```diff
@@ -345,6 +346,9 @@ def validate(
         4         0.80      dog
         5         0.76      dog
         """
+        if not CONFIG.validation_enabled:
+            return
```

Comment:
this should return the original dataframe, as a call to `schema.validate` must still return a dataframe.